index.html

```diff
@@ -99,6 +99,9 @@ <h2>Stats</h2>
 
     <div id="settings-screen" slot="screen">
       <h2>Settings</h2>
+      <address>
+        Built by <a href="https://playpass.games">Playpass Games</a>
```

Comment:
Suggested tweak for these:

```suggestion
        Built with <a href="https://playpass.games" target="_blank">Playpass</a>
```

`by --> with` because this messaging will end up in games created off the templates that are created by external developers 😃